### ISSUE TITLE
Dropdown.js doesn't seem to work on Heroku

### ISSUE THREAD

**Issue by emrysgraefe** [anjlab/bootstrap-rails#85]
I upgraded from Bootstrap 2.3.2 and now I can't seem to get the dropdown menu javascript to work anymore.  I've tried precompiling assets, moving the order of resources in application.js, using the rails_12factor gem and ensuring that the config file options are correct.

**Comment by JanStevens** [comment]
I still have the same problem, after a page change the drop down does not work anymore (something to do with turbo links I guess)

**Comment by yury** [comment]
@JanStevens  do you use https://github.com/kossnocorp/jquery.turbolinks ?

**Comment by JanStevens** [comment]
Yes I Use it, I now disabled turbolinks in my layout and ofcourse now it works,
